zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -898,6 +898,42 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+
+  /messages/pubblication:
+    post:
+      tags:
+        - Message
+      summary: Publish a message
+      description: |
+        Publishes a single message. Messages are published to specific partitions computed from
+        their correlation keys.
```

```suggestion
        Publishes a single message. 
        Messages are published to specific partitions computed from their correlation keys.
```

:x: Endpoint `description`s require a complete sentence on the first line.